src/partials/contacts.html

```diff
@@ -0,0 +1,63 @@
+<section class="contacts">
+  <div class="contacts__chicago">
+    <span>CAFE</span>
+    <h4>Chicago</h4>
+    <p>Fusce ut velit laoreet, tempus arcu molestie vulputate</p>
+    <ul>
+      <li>
+        <h4>Monday - Friday</h4>
+        <p>06:00 AM - 10:00 PM</p>
```

Comment:
Do użycia znacznik <time>10:00</time>

Znacznik <time>
Datę i godzinę można zapisać zwykłym tekstem, ale lepiej jest użyć specjalnego znacznika <time>. Data dla osoby to tekst wewnątrz znacznika, a dla maszyny - wartość atrybutu datetime w formacie [ISO 8601](https://ru.wikipedia.org/wiki/ISO_8601).

<p>
  Najnowsze wiadomości <time datetime="2019-12-14">14 grudnia 2019 roku</time>.
</p>

<p>
  Spotkanie odbyło się o <time datetime="2018-07-14T11:30">11:30 rano</time>.
</p>

Przeglądarka wyświetli tylko tekst wewnątrz znacznika. Osoba zobaczy zrozumiały tekst, a technologie asystujące odczytają atrybut datetime i uzyskają dokładną wartość w wymaganym formacie.